commands/commands_test.go

```diff
@@ -1,11 +1,19 @@
 package commands
 
 import (
+	"io/ioutil"
+	"os"
 	"testing"
 
 	"github.com/github/hub/Godeps/_workspace/src/github.com/bmizerany/assert"
+	"github.com/github/hub/console"
 )
 
+func TestMain(m *testing.M) {
+	console.Default = console.Console{Stdout: ioutil.Discard, Stderr: ioutil.Discard}
```

Comment:
I did some refactoring again. This is the new code to mute the print out to console in tests